### DISCUSSION TITLE
When using external data source there is now "Create View"-Button

### DISCUSSION THREAD

**Discussion by gandalf75** [Budibase/budibase#4828] [Ideas]
**Describe the bug**
I wanted to use an external data source and create views for existing tables.

I managed to connect to an existing data source (MySQL). However, after doing so the "Create view"-Button does not exist.

In comparison, when using "Budibase DB", there is a "Create view"-button.

I could not find any information for this behavior in the user documentation (I looked [here](https://docs.budibase.com/building-apps/data/views))

**To Reproduce**
1.) Create external MySQL source as described in https://docs.budibase.com/building-apps/data/tables/postgresql-mysql-and-sql-server
2.) Click on one of the imported tables

**Expected behavior**
When connecting to an external data source it is possible to create a view in the same way as when using an internal Budibase DB

**Screenshots**

When using external MySQL data source:
![grafik](https://user-images.githubusercontent.com/48381748/157042849-f99b1531-e1cb-4821-9a97-300b0ddfe59f.png)

When using internal Budibase DB:
![grafik](https://user-images.githubusercontent.com/48381748/157043261-cda29c47-7a9d-4bcd-8ca1-dbff27379458.png)

**Desktop (please complete the following information):**
 - Windows 10
 - Firefox, Google Chrome

**Comment by mike12345567** [comment]
Hi @gandalf75 - this is something that Budibase doesn't support at this current time, I've converted this to a discussion so that it can be marked as an idea for the future.

**Reply by gandalf75** [reply]
Ok, thanks for your quick reply @mike12345567 !
